src/components/page-elements/layout/layout.js

```diff
@@ -2,11 +2,11 @@ import React from 'react'
 import { Main } from './layout-style'
 import { Nav } from 'components/page-elements'
 
-export const Layout = ({ children }) => {
+export const Layout = ({ is_dark_background, children }) => {
```

Comment:
is_static ?

Reply:
`is_static_page` is not really descriptive. Since this props only control the logo, can we name it `is_dark_logo` instead of `is_dark_background`?